src/modules/ilos/infrastructure/decorators/ilo-permission.decorator.spec.ts

```
@@ -0,0 +1,78 @@
+  it('should apply the decorator to a class', () => {
+    const mockSetMetadata = SetMetadata as jest.Mock;
+    mockSetMetadata.mockReturnValue((target: any) => {
+      if (!target.metadata) {
+        target.metadata = {};
+      }
+      target.metadata['iloPermission'] = IloPermissionType.CONTROL_ILO_POWER;
+      return target;
+    });
+
+    @IloPermission(IloPermissionType.CONTROL_ILO_POWER)
+    class TestController {
```

ESLint

[error] 'TestController' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)